Source/Public/New-ADOPSGroupEntitlement.ps1

```diff
@@ -0,0 +1,84 @@
+function New-ADOPSGroupEntitlement {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [string]$GroupOriginId,
+
+        [Parameter()]
```

Comment:
This parameter isn't used. Im guessing it was moved to a hard coded value, which is 100% ok, but please remove the param if it is not needed any more :)

Reply:
@Hardstl This was resolve but unchanged.. Is there a reason to keep it as a parameter without using it?